webpack/assets/javascripts/react_app/components/common/forms/Select.js

```diff
@@ -17,7 +17,7 @@ class Select extends React.Component {
     const renderOptions = arr =>
       map(arr, (attribute, value) => (
         <option key={attribute} value={value}>
```

Comment:
nitpick: could you assign `value` to the key prop?

Reply:
Are you sure, that setting "key=value" is correct? (BTW, could not find the purpose of "key" in https://www.w3schools.com/tags/ref_attributes.asp - may be react specific?)